e2e/analysis/analysis_build_locally_test.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2021 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package analysis_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/ZupIT/horusec-devkit/pkg/utils/logger"
+	"github.com/ZupIT/horusec/config"
+	"github.com/ZupIT/horusec/e2e/analysis"
+	customimages "github.com/ZupIT/horusec/internal/entities/custom_images"
+	"github.com/ZupIT/horusec/internal/utils/testutil"
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"os"
+	"path/filepath"
+)
+
+var _ = Describe("Run a complete horusec analysis when build tools locally", func() {
+	tmpDir := CreateHorusecConfigAndReturnTMPDirectory()
+
+	Describe("Running e2e tests when build tools locally", func() {
+		allTestsCases := analysis.NewTestCase(tmpDir)
+		for idx := range allTestsCases {
+			testCase := allTestsCases[idx]
+			logger.LogInfo("Preparing test for run e2e build locally: ", testCase.Language, fmt.Sprintf("[%v/%v]", idx + 1, len(allTestsCases)))
+			session, err := testCase.RunAnalysisTestCase()
+			if err != nil {
+				Fail(fmt.Sprintf("Error on run analysis Test Case when build: %v", err))
+			}
+			session.Wait(testutil.AverageTimeoutAnalyzeForExamplesFolder)
+			testCase.OutputAnalysis = string(session.Out.Contents())
+			testCase.ExitCodeAnalysis = session.ExitCode()
+
+			It(fmt.Sprintf("execute command without error for: %s", testCase.Language), func() {
```

Comment:
i think there is no need for all this `It` statements, we could assert everything in one it, if there is a need to describe we can use the [By](https://onsi.github.io/ginkgo/#documenting-complex-its-by) operator

Reply:
sounds good for me